docs/devguide/contributing.asciidoc

```diff
@@ -102,7 +102,7 @@ recommend that you install it.
 === Update scripts
 
 The Beats use a variety of scripts based on Python, make and mage to generate configuration files
-and documentation. The primary command used for this is:
+and documentation. Ensure to use the version of python listed in the https://github.com/elastic/beats/blob/main/.python-version[.python-version] file. The primary command used for this is:
```

Comment:
I tried to read the entire section and it reads like "the primary command for setting the Python version is...".

I think we should also change the next sentence:

```suggestion
and documentation. Ensure to use the version of python listed in the https://github.com/elastic/beats/blob/main/.python-version[.python-version] file. 

The primary command for updating generated files is:
```